cypress/e2e/client-tasks/pagination.cy.js

```diff
@@ -1,93 +1,15 @@
 describe("Pagination", () => {
-  beforeEach(() => {
+  before(() => {
     cy.setCookie("Other", "other");
     cy.setCookie("XSRF-TOKEN", "abcde");
```

Comment:
simplified this to match all the other pagination tests, must have been missed before